packages/node_modules/@webex/webex-core/test/integration/spec/credentials/credentials.js

```diff
@@ -19,6 +19,25 @@ describe('webex-core', () => {
         user = u;
       }));
 
+    describe('#config', () => {
+      let webex;
+
+      it('should accept an authorizationString to set authorizeUrl', () => {
+        webex = new WebexCore({
+          config: {
+            credentials: {
+              authorizationString: 'https://api.example.com/v1/auth?example=value'
+            }
+          }
+        });
+
+        assert.include(
+          webex.config.credentials.authorizationString,
+          webex.config.credentials.authorizeUrl
+        );
```

Comment:
Could you assert that the `authorizeUrl` is set to the value we expect? In this case it should be `https://api.example.com/v1/auth`.

Reply:
I've updated this based on the comments.